app/user.go

```diff
@@ -1466,6 +1466,10 @@ func (a *App) PermanentDeleteUser(user *model.User) *model.AppError {
 		return err
 	}
 
+	if err := a.Srv.Store.Bot().PermanentDelete(user.Id); err != nil {
+		return err
```

Comment:
Just to confirm. This doesn't error out if the user isn't a bot right?

Reply:
@crspeller 	Correct.  I tested and the `PermanentDelete()` Bot store method returns `nil` if the user doesn't exist.